packages/volto/news/7075.feature

```diff
@@ -0,0 +1,4 @@
+Add docs for writing a new widget.
+Work on the related file to respect the Plone docs writing guidelines.
+Add {file} tags to inline code sections that refer to file paths.
```

Comment:
Apologies for the pedantry, but FYI https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html#roles-an-in-line-extension-point.

```suggestion
Add `{file}` role to sections that refer to file paths.
```